extensions/grpc/inprocess/pom.xml

```diff
@@ -17,5 +17,9 @@
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-grpc</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-inprocess</artifactId>
```

Comment:
They extracted this into new / diff jar?

Reply:
Yes…